### PR TITLE
[Excel] (Custom functions) Minor updates to Dialog page RE: new d.ts

### PR DESCRIPTION
**Pull request by mscharlock**
Minor changes to Dialog sample to reflect new template and d.ts info

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[080b29b](https://github.com/OfficeDev/office-js-docs-pr/pull/934/commits/080b29b9897f08ddcdd968f7c545974a1ef5d126)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-dialogupdate/docs/excel/custom-functions-dialog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904290530497080-934%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=EsvY%2FWCdH%2FqWgOMryuSpDNCXnv9h8JM5iZ9A%2FpsLL1M%3D&st=2019-04-29T05%3A27%3A00Z&se=2019-05-30T05%3A32%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-dialog.md [hunk 1]
```diff
@@ -1,22 +1,25 @@
 ---
-ms.date: 03/21/2019
+ms.date: 034/28/2019
```
Comment (AlexJerabek) [inline docs/excel/custom-functions-dialog.md]: Typo "034" -> "04"

File: docs/excel/custom-functions-dialog.md [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 description: Create dialog boxes through custom functions in Excel using JavaScript.
 title: Custom functions dialogs (preview)
 localization_priority: Priority
 ---
 # Display a dialog box in custom functions
 
-If your custom function needs to interact with the user, you can create a dialog box using the `OfficeRuntime.Dialog` object. A common scenario for using the dialog box is to authenticate a user so that your custom function can access a web service. For more details about authentication with custom functions, see [Custom functions authentication](./custom-functions-authentication.md).
+If your custom function needs to interact with the user, you can create a dialog box using the `Office.Dialog` object. A common scenario for using the dialog box is to authenticate a user so that your custom function can access a web service. For more details about authentication with custom functions, see [Custom functions authentication](./custom-functions-authentication.md).
```
Comment (AlexJerabek) [inline docs/excel/custom-functions-dialog.md]: Can we change the first reference to Office.Dialog to a link to the Dialog object reference page?

File: docs/excel/custom-functions-dialog.md [hunk 3]
```diff
@@ -72,13 +75,13 @@ function getStock (ticker) {
         }, 1000);
       } else {
         _dialogOpen = true;
-        OfficeRuntime.displayWebDialog(url, {
+        Office.displayWebDialogOptions(url, {
```
Comment (AlexJerabek) [inline docs/excel/custom-functions-dialog.md]: This doesn't match the [current ref doc page](https://docs.microsoft.com/en-us/javascript/api/office-runtime?view=office-js) Are the reference docs out of date?

Comment (mscharlock) [reply docs/excel/custom-functions-dialog.md]: Yes, they will be updated later when the d.ts goes live (which it theoretically is but I need to check with Joe/Madhav)

File: docs/excel/custom-functions-dialog.md (hunk 2 above)
Comment (mscharlock) [reply docs/excel/custom-functions-dialog.md]: Good idea!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[92592db](https://github.com/OfficeDev/office-js-docs-pr/pull/934/commits/92592dbc1bddc7efe64917e2e8ab9e823097f991)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-dialogupdate/docs/excel/custom-functions-dialog.md) | :white_check_mark:Succeeded | |
[javascript/api/office-runtime/officeruntime.dialog](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-dialogupdate/javascript/api/office-runtime/officeruntime.dialog) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904291849266950-934%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=m1hNoIHPkjgFQ6LEYbYr8xexEsq2jMkhaEIuz6ikKAk%3D&st=2019-04-29T18%3A45%3A40Z&se=2019-05-30T18%3A50%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mscharlock** [comment]
Note to self ok to merge when d.ts goes live

**Comment by mscharlock** [comment]
Merging to master for bug bash, note that d.ts is not up to date yet

**Pull request merged**